File: src/com/java08_java_unit_test/unit_test/src/test/java/unit/test/CalculatorTest.java
package unit.test;

import static org.junit.jupiter.api.Assertions.*;
import org.junit.jupiter.api.Test;


public class CalculatorTest {
    private Calculator calculator = new Calculator();

    @Test
    public void addTest() {
        var result = calculator.add(10, 10);
        assertEquals(20, result);
    }
}

File: src/kotlin06_unit_testing/app/src/test/kotlin/lesson03_membuat_test/CalculatorTest.kt
package lesson03_membuat_test

import kotlin06_unit_testing.Calculator
import org.junit.jupiter.api.Test

class CalculatorTest {

    val calculator: Calculator = Calculator()

    @Test
    fun testAddSuccess(): Unit {
        val result = calculator.add(10, 10)
    }

    @Test
    fun testAddSuccess2(): Unit {
        val result = calculator.add(10, 10)
    }

}